### PR TITLE
NETOBSERV-872: Add netobserv ci-job

### PR DESCRIPTION
**Pull request by Amoghrd**

### CODE REVIEW

**Comment by Amoghrd** [comment]
Could you PTAL when you get the chance. TIA! @dhellmann @bparees @jmguzik @jupierce

File: core-services/prow/02_config/netobserv/network-observability-operator/_pluginconfig.yaml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 approve:
 - commandHelpLink: ""
-  ignore_review_state: true
   repos:
   - netobserv/network-observability-operator
-  require_self_approval: true
+  require_self_approval: false
```
Comment (bparees) [inline core-services/prow/02_config/netobserv/network-observability-operator/_pluginconfig.yaml:5]: why is this change part of this PR?

File: ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  to: network-observability-operator
+operator:
+  bundles:
+  - as: noo-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/netobserv/network-observability-operator:1.0.4
+    with: network-observability-operator
+promotion:
+  name: "4.14"
```
Comment (bparees) [inline ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml:23]: why 4.14?

**Review by bparees — CHANGES_REQUESTED** [review]
in addition to my comments:

1) this is adding a CI job for a new repo, it's not "adding it to cluster-bot" so please fix the PR title/commit text
2) please compact/squash your commits.  It's ok to keep separate commits for unrelated changes(if they really don't belong in a separate PR), or for separating generated code from manually edited code, but the commits on this PR don't seem to be structured that way, just iterative changes.

File: ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml (hunk 2 above)
Comment (Amoghrd) [reply ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml:23]: changed to 4.15

Comment (bparees) [reply ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml]: master/main represents 4.16 currently i believe.

Comment (bparees) [reply ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml]: see: https://github.com/openshift/release/blob/3d560d554670c6f6eb60ab9aa593a93e7af6f908/ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml#L29-L32

Comment (Amoghrd) [reply ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml]: ah okay got it. Will update that

**Comment by Amoghrd** [comment]
Closing in favor of https://github.com/openshift/release/pull/47683

**Pull request closed without merging**